apps/web/public/static/locales/en/common.json

```diff
@@ -1599,7 +1599,8 @@
   "attendee_email_variable": "Attendee email",
   "attendee_email_info": "The person booking's email",
   "kbar_search_placeholder": "Type a command or search...",
-  "invalid_credential": "Oh no! Looks like permission expired or was revoked. Please reinstall again.",
+  "invalid_credential": "looks like permissions/credentials expired or were revoked for",
```

Comment:
This should be "It looks like permissions expired or were revoked for {{appName}}."